internal/armada/repository/job_test.go

```diff
@@ -259,24 +205,6 @@ func TestDeleteWithSomeMissingJobs(t *testing.T) {
 	})
 }
 
-func TestReturnLeaseForDeletedJobShouldKeepJobDeleted(t *testing.T) {
```

Comment:
Does this Test still pass with your changes?

Most likely this was added as previously I imagine that wasn't the case

Reply:
Since making returning lease for deleted jobs a no-op, this passes.